UPGRADING.rst

```diff
@@ -11,6 +11,39 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#2246 Add deployment incarnation counter
+========================================
+
+See upgrading instructions for #2143 below.
+
+
+#2143 Merge service accounts for indexer and service
+============================================================
+
+1.  Before upgrading to this commit, run ::
+
+        make -C terraform validate
+        (cd terraform && terraform destroy -target=google_service_account.indexer)
+
+2.  Upgrade to this commit or a later one.
+
+3.  Let someone who can administer the SAM group that controls access to TDR
+    know of the rename for the service account email via Slack. The
+    administrator will need to replace the old service account email with the
+    new one. For example, ::
```

Comment:
I don't think we should use a literal block for this. It looks odd:

![image](https://user-images.githubusercontent.com/5143256/95926671-b25f7d00-0d71-11eb-9580-c1aeed1c0088.png)

Reply:
I think following the same pattern that was used in [Section 1645](https://github.com/DataBiosphere/azul/blob/issues/amar/2143-merge-service-accounts/UPGRADING.rst#1645-rethink-template-config-variable-mechanism)  would work here. Need to see how github renders the changes.

Reply:
![Screen Shot 2020-10-15 at 9 26 41 AM](https://user-images.githubusercontent.com/1947534/96158610-8acef880-0ec8-11eb-800f-c0cde2a208f0.png)